handsontable/src/plugins/contextMenu/predefinedItems/alignment.js

```diff
@@ -4,6 +4,53 @@ import * as C from '../../../i18n/constants';
 
 export const KEY = 'alignment';
 
+/**
+ * @param  {string} htClassName The class name to check.
+ * @returns {boolean} Returns `true` if the cell has the provided class name.
+ */
+function getAlignmentComparator(htClassName) {
```

Comment:
Can I suggest moving all additional helpers/utils functions to the `utils.js` file? I think it'll be consistent with other menu functions. 

Also, as you created that helper function can I ask you to replace all repetitive code, such as
```js
const hasClass = checkSelectionConsistency(this.getSelectedRange(), (row, col) => {
  const className = this.getCellMeta(row, col).className;

  if (className && className.indexOf('htLeft') !== -1) {
    return true;
  }
});
```
within all file?

Reply:
Fixed. `getAlignmentComparator` moved to the `utils` and renamed to the `getAlignmentComparatorByClass`. Also, reused for all alignments